### DISCUSSION TITLE
Facing Multisig Issue

### DISCUSSION THREAD

**Discussion by Prikshit-Kumar** [cosmos/community#53] [Chain Builders (Cosmos SDK)]
Hii there! 
I am working on the Cosmos Project and trying to submit the transaction through multisig Account. 
But getting an error: Signature Verification Failed! 
In outdated versions of Cosmos, Ignite, and Go, everything was working. But after the version upgrade, started facing this issue. 

**Error:** signature verification failed; please verify account number (127), sequence (0) and chain-id (cn-chaincode-fi): unauthorized

I tried some methods, like Proto's updates, and message verification through encoding and decoding, messages seem to be the same. But didn't get the chance to dig into Cosmos Function Definitions.

**Latest versions:**
Ignite - v0.27.1
Cosmos SDK - v0.47.3
Go - v1.21.1

**Comment by tac0turtle** [comment]
Hey were you using ledger or normal cli signing?
